app/views/account/pages/show.html.slim

```diff
@@ -0,0 +1,21 @@
+.container
+  h1= @page.title
+  .row
+    h3 Page type:
+    p= @page.page_type
```

Comment:
invalid usage of bootstrap grids

Reply:
yes. 
col-md-12 needs to be added
and h1 also needs to be wrapped.

THIS IS GLOBAL PROBLEM FOR ALL OF YOU

Just try to discuss bootstrap together and understand how it works. We don't want to explain it to everyone.